libs/remix-ui/editor/src/lib/remix-ui-editor.tsx

```diff
@@ -288,7 +290,7 @@ export const EditorUI = (props: EditorUIProps) => {
 
   props.editorAPI.getFontSize = () => {
     if (!editorRef.current) return
-    return editorRef.current.getOption(42).fontSize
+    return editorRef.current.getOption(43).fontSize
```

Comment:
What does 43 mean?

Reply:
is the option for the fontsize. because I am using a later version of monaco the option nr changed

Reply:
now we can use the enum from types... I will change it